lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php

```diff
@@ -219,6 +225,17 @@ public function setTemplateOptions($templateOptions)
         return $this;
     }
 
+    /**
+     * Set store
+     * @param \Magento\Store\Model\Store
+     * @return $this;
+     */
+    public function setStore($store)
```

Comment:
Type hint for `\Magento\Store\Api\Data\StoreInterface` would be a nice addition here